UPGRADING.md

```diff
@@ -2,6 +2,357 @@
 
 This guide provides detailed instructions on how to upgrade between major versions of the Chat SDK.
 
+## 0.14.x to 0.15.0
+
+### Typing and Occupancy Current Changed to Properties
+
+**Expected Impact: High**
```

Comment:
I think this would be low/medium tops - it's not a huge breaking change requiring a lot of work, it's just a find and replace